<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MHLFileVerify</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../share/icons/app_logo.png</normaloff>../share/icons/app_logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(28, 28, 32);
selection-color: rgb(246, 97, 81);
font: 11pt &quot;Open Sans Regular&quot;;
color: rgb(200,200,200);
selection-background-color: rgb(20,20,20);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgb(33, 33, 38);
selection-color: rgb(36, 31, 49);
selection-background-color: rgb(198, 70, 0);
}

QMessageBox {
background-color: rgb(28, 28, 32);
selection-color: rgb(246, 97, 81);
font: 9pt &quot;Open Sans Light&quot;;
selection-background-color: rgb(20,20,20);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Verify">
       <attribute name="title">
        <string>Verify MHL file</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,2,1">
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_icon_1">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="icon_mhl_file_label">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel:hover {
    background-color: #28282e;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../share/icons/mhl_document.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_text_1">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Choose or drag a compatible MHL file:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_edit_file">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_buttons_1">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>40</number>
           </property>
           <property name="rightMargin">
            <number>40</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>30</number>
             </property>
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QPushButton" name="button_open_file">
               <property name="text">
                <string>Open file</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_verify">
               <property name="text">
                <string>Verify</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Create">
       <attribute name="title">
        <string>Create MHL file</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="3,2,1">
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_icon_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="icon_drive">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel:hover {
    background-color: #28282e;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../share/icons/media_128.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_text_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Choose or drag a dir to seal:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="line_edit_directory">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_checksum">
             <property name="currentText">
              <string>md5</string>
             </property>
             <item>
              <property name="text">
               <string>md5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>sha1</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Checksum</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_buttons_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>40</number>
           </property>
           <property name="rightMargin">
            <number>40</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>30</number>
             </property>
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QPushButton" name="button_choose_dir">
               <property name="text">
                <string>Choose</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_create">
               <property name="text">
                <string>Create</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_mhl_file"/>
    <addaction name="actionVerify"/>
    <addaction name="actionCreate_mhl_file"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShortcuts"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShortcuts">
   <property name="icon">
    <iconset>
     <normaloff>../share/icons/input-keyboard.png</normaloff>../share/icons/input-keyboard.png</iconset>
   </property>
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../share/icons/help-about.png</normaloff>../share/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_mhl_file">
   <property name="icon">
    <iconset>
     <normaloff>../share/icons/folder-open.png</normaloff>../share/icons/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>Open MHL file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>../share/icons/application-exit.png</normaloff>../share/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionVerify">
   <property name="icon">
    <iconset>
     <normaloff>../share/icons/sync.png</normaloff>../share/icons/sync.png</iconset>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionCreate_mhl_file">
   <property name="icon">
    <iconset>
     <normaloff>../share/icons/document-new.png</normaloff>../share/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Create MHL file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
